_layouts/default.html

```diff
@@ -49,31 +49,32 @@
         <a href="/"><img class="{{ logo_class }}" src="/assets/compiler_logo_stacked.svg" alt="Compiler logo"></a>
     </header>
 
-    <main>
+    <main class="container-fluid">
         {{ content }}
     </main>
 
-    <footer>
-        <div class="divider-line"></div>
-        <div class="certs d-flex flex-column flex-lg-row row-cols-lg-4">
-            <div>
-                <a href="mailto:hello@compiler.la" class="monospace contact-link">hello@compiler.la</a>
-                <p class="address">
-                    525 S Hewitt St<br />Los Angeles, CA 90013
-                </p>
-            </div>
+    <footer class="border-top border-light">
+        <div class="container-fluid">
+            <div class="row d-flex flex-column flex-lg-row row-cols-lg-4">
+                <div class="pt-lg-5 mt-lg-1 pt-4 mt-3 pb-5">
+                    <a href="mailto:hello@compiler.la" class="monospace text-primary contact-link">hello@compiler.la</a>
```

Comment:
This `text-primary` class gets its color from this CSS Variable declaration:

```css
    --bs-primary: #88B440; /* Use with .text-primary */
    --bs-primary-rgb: 136, 180, 64;
```

Can also be used with `border-primary`.